ckanext/unaids/plugin.py

```diff
@@ -71,7 +71,7 @@ class UNAIDSPlugin(p.SingletonPlugin, DefaultTranslation):
     This plugin implements the configurations needed for AIDS data exchange
 
     """
-
+    resources_to_validate_package = {}
```

Comment:
This looks broken. Are you sure that it safe to use this dict like that? `UNAIDSPlugin` is a singleton plugin. I feel like `resources_to_validate_package` is going to be shared across all calls to CKAN after it's initialized once during startup.

I've seen your comment and took a look at the implementation in ckanext-validation. Let's hope that it's going to work as advertised ;)
Can you move this declaration to below `# IResourceController` so it's clearer on where this things is needed and used?